inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/enums/BizConstant.java

```diff
@@ -24,6 +24,8 @@
 
     public static final String STORAGE_HIVE = "HIVE";
 
+    public static final String STORAGE_CLICKHOUSE = "CLICKHOUSE";
```

Comment:
there are several storage classes, can we manage them using a `storage enum`?

Reply:
When the function is written, optimize it together next time